components/com_publications/site/controllers/publications.php

```diff
@@ -470,6 +475,25 @@ public function browseTask()
 		// Run query with limit
 		$results = $model->entries('list', $filters);
 
+		// Lee: Check that publication has an attachment
```

Comment:
Is prefixing a comment with the developer's name typical for this codebase?

If we're tagging this code as local modifications, then I would prefer "UVic" or "ARCsoft" as appropriate--`git blame` can always be used to determine the individual developer.

If this is a reminder to check something again, `TODO` or `TODO(drew):` is the way I would go.

Reply:
I don't see any reason to have that or `Lee:` in this line at all.  Please take it out or clarify.

Reply:
Moving forward `TODO` is appropriate. Actually I should have asked if @ghjensen requested we do this to mark off sections we've modified--is this the case?

If not, I'd say we use `TODO` if it's a TODO, and otherwise leave out these labels unless/until we have a reason to use them.